packages/orchestration/src/exos/cosmos-orchestration-account.js

```diff
@@ -291,26 +291,29 @@ export const prepareCosmosOrchestrationAccountKit = (
      * @param {RemoteIbcAddress} info.remoteAddress
      * @param {object} io
      * @param {IcaAccount} io.account
-     * @param {Remote<StorageNode>} io.storageNode
+     * @param {Remote<StorageNode> | undefined} io.storageNode
```

Comment:
key should also be optional?
```suggestion
     * @param {Remote<StorageNode>} [io.storageNode]
```